web/vtadmin/src/components/routes/tablet/Advanced.tsx

```diff
@@ -0,0 +1,242 @@
+                                <Icon icon={Icons.alertFail} className="fill-current text-danger inline mr-2" />
+                                Command StartTablet cannot be run on the primary tablet.
+                            </p>
+                        )}
+                        <button
+                            onClick={onStartReplication}
+                            className="btn btn-secondary mt-4"
+                            disabled={primary || startReplicationLoading}
+                        >
+                            Start replication
+                        </button>
+                    </div>
+                    <div className="p-8">
+                        <div className="flex justify-between items-center">
+                            <p className="text-base font-bold m-0 text-gray-900">Stop Replication</p>
+                            <a
+                                href="https://vitess.io/docs/reference/programs/vtctl/tablets/#stopreplication"
+                                target="_blank"
+                                rel="noreferrer"
+                                className="text-gray-900 h-6 w-6 ml-1"
+                            >
+                                <Icon icon={Icons.open} className="text-gray-900 fill-current" />
+                            </a>
+                        </div>
+                        <p className="text-base m-0">
+                            This will run the underlying database command to stop replication on tablet{' '}
+                            <span className="font-bold">{alias}</span>. For example, in mysql 8, this will be{' '}
+                            <span className="font-mono text-sm p-1 bg-gray-100">stop replication</span>.
+                        </p>
+                        {primary && (
+                            <p className="text-danger">
+                                <Icon icon={Icons.alertFail} className="fill-current text-danger inline mr-2" />
+                                Command StopTablet cannot be run on the primary tablet.
+                            </p>
+                        )}
+                        <button
+                            onClick={onStopReplication}
+                            className="btn btn-secondary mt-4"
+                            disabled={primary || stopReplicationLoading}
+                        >
+                            Stop replication
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div className="my-8">
+                <h3 className="mb-4">Reparent</h3>
+                <div className="w-full border rounded-lg border-gray-400">
+                    <div className="p-8 border-b border-gray-400">
+                        <div className="flex justify-between items-center">
+                            <p className="text-base font-bold m-0 text-gray-900">Reparent Tablet</p>
+                            <a
```

Comment:
Thank youuuu for linking to the docs, this is amazing!!! 😍 

What do you think about adding a little "documentation" (or "docs" or "reference" or something) string adjacent to the icon to make the link super obvious? I could totally seem myself missing (but wanting!) that link if I'm in VTAdmin at 2am during an incident or something.

Reply:
For sure! :-)